linux/mcompile/keymap/mcompile.cpp

```diff
@@ -521,44 +525,34 @@ KMX_BOOL KMX_DoConvert(LPKMX_KEYBOARD kbd, KMX_BOOL bDeadkeyConversion, gint arg
 }
 
 /**
- * @brief  return an array of [usvk, ch_out] pairs: all existing combinations of a deadkey + character for the underlying keyboard
+ * @brief  return a vector of [usvk, ch_out] pairs: all existing combinations of a deadkey + character for the underlying keyboard
  * @param      dk_Table    shiftstate of the deadkey
  * @param      deadkey     deadkey character
- * @param[out] outputPairs pointer to array of [usvk, ch_out] pairs
+ * @param[out] dk_vec vector of [usvk, ch_out] pairs
  * @param      keymap      pointer to  the currently used (underlying) keyboard Layout
  * @return size of array of [usvk, ch_out] pairs
```

Comment:
```suggestion
 * @return size of vector of [usvk, ch_out] pairs
```

Although I think it's triplets, not pairs.